example-playbooks/restore_scylla_manager_backup/restore.yaml

```diff
@@ -0,0 +1,135 @@
+---
+
+- name: Restore tokens
+  hosts: all
+  become: true
+  gather_facts: false
+  tasks:
+    - name: Stop Scylla service
+      service:
+        name: scylla-server
+        state: stopped
+
+    - name: Remove Scylla directories
+      file:
+        path: "/var/lib/scylla/{{ item }}"
+        state: absent
+      loop:
+        - data
+        - commitlog
+        - hints
+        - view_hints
+
+    - name: Recreate Scylla directories
+      file:
+        path: "/var/lib/scylla/{{ item }}"
+        state: directory
+        owner: scylla
+        group: scylla
+        mode: 0755
+      loop:
+        - data
+        - commitlog
+        - hints
+        - view_hints
+
+    - name: Get seeds list
+      shell: |
+        grep '\- seeds:' /etc/scylla/scylla.yaml | awk '{ print $NF }' | tr -d '"' | tr -d "'"
+      register: _seeds
+
+    - name: Get initial token
+      shell: |
+        scylla-manager-agent download-files -L {{ backup_location }} -n {{ host_id[inventory_hostname] }} -T {{ snapshot_tag }} --dump-tokens
+      register: _initial_token
+
+    - name: Set initial_token in scylla.yaml
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^(#\s*)?initial_token:'
+        line: "initial_token: {{ _initial_token.stdout }}"
+        create: yes
+
+    - name: Make each node a seed
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^(\s+) - seeds:'
+        line: |
+          \g<1> - seeds: {{ groups.all | join(',') }}
+        backrefs: yes
+
+    - name: Start Scylla service
+      service:
+        name: scylla-server
+        state: started
+
+    - name: Wait for CQL port
+      wait_for:
+        port: 9042
+
+    - name: Stop Scylla service
+      service:
+        name: scylla-server
+        state: stopped
+
+- name: Restore schema
+  hosts: all[0]
+  become: true
+  gather_facts: false
+  tasks:
+    - name: Remove other seeds
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^(\s+) - seeds:'
```

Comment:
You want to set `node0` as a seeder to other nodes in the cluster too - not just for itself.

Reply:
I want to restore schema on single node, start it and then let the other node pull it.
I eventually restore original seed nodes.
I renamed it to `Temporarily set as the only seed`